scripts/debug-env-loading.js

```
@@ -0,0 +1,140 @@
+      'QSTASH_NEXT_SIGNING_KEY exists after dotenv:',
+      !!process.env.QSTASH_NEXT_SIGNING_KEY,
+    );
+  }
+} catch (error) {
+  console.log('❌ Exception loading dotenv:', error.message);
+}
+
+// Try loading with dotenv and specific path
+console.log('\n3. LOADING WITH SPECIFIC ENV FILE PATH:');
+console.log('-----------------------------------------');
+const fs = require('fs');
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)